src/status_im/ui/screens/syncing/sheets/sync_device_notice/views.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.ui.screens.syncing.sheets.sync-device-notice.views
+  (:require
+   [quo.react-native :as rn]
+   [status-im.ui.screens.syncing.sheets.sync-device-notice.styles :as styles]
+   [status-im.ui.screens.syncing.sheets.enter-password.views :as enter-password]
+   [re-frame.core :as re-frame]
+   [status-im.ui.components.react :as react]
+   [quo2.components.buttons.button :as quo2-button]
+   [status-im.utils.handlers :refer [>evt]]
+   [status-im.react-native.resources :as resources]
+   [status-im.i18n.i18n :as i18n]))
+
+(defn hide-sheet-and-dispatch [event]
+  (re-frame/dispatch [:bottom-sheet/hide])
+  (re-frame/dispatch event))
+
+(defn views []
+  [:<>
+   [rn/view {:style styles/sync-devices-header}
+    [react/image {:source (resources/get-image :sync-new-device)
+                  :style styles/sync-devices-header-image}]]
+   [rn/view {:style styles/sync-devices-body-container}
+    [rn/text {:style styles/header-text} (i18n/label :t/privacy-policy)]
+    [rn/text {:style styles/instructions-text} (i18n/label :t/sync-instructions-text)]
+    [rn/text {:style styles/list-item-text} (i18n/label :t/sync-instruction-step-1)]
+    [rn/text {:style styles/list-item-text} (i18n/label :t/sync-instruction-step-2)]
+    [rn/text {:style styles/list-item-text} (i18n/label :t/sync-instruction-step-3)]
+    [quo2-button/button {:type   :secondary
+                         :size    40
+                         :style   styles/setup-syncing-button
+                         :before  :main-icons2/face-id20
+                         :on-press #(>evt [:bottom-sheet/show-sheet
```

Comment:
- [Suggestion] `quo2-button/button` seems excessive, just aliasing as `button` is enough.
- [Guidelines] Use `:i` qualification instead of `:main-icons2`.
- [Guidelines] Use `utils.re-frame/dispatch` instead of `>evt`.